app/components/ProfilePicture.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Avatar = styled.img`
+    width: inherit;    
```

Comment:
We should probably set maxWidth here so the image isn’t displayed too huge. Same with min maybe